hil/ext/auth/database.py

```diff
@@ -60,6 +61,19 @@ def set_password(self, password):
                          db.Column('project_id', db.ForeignKey('project.id')))
 
 
+@rest_call('GET', '/auth/basic/user', schema=Schema({}))
```

Comment:
Usually our standard is to have a plural url for listing all of something, i.e. :
`@rest_call('GET', '/nodes/<is_free>', Schema({'is_free': basestring}))`

What do we think about having the url be `/auth/basic/users`?

Reply:
That's probably better for consistency's sake, yeah.